css/style.css

```
@@ -202,5 +240,130 @@ footer {
 .footer__sns__image {
   width: 2rem;
   height: 2rem;
-  color: #ffffff;
+  color: var(--color-white);
+}
+
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+  nav {
+    padding: 0rem 3.2rem;
+  }
+
+  .nav__container {
+    padding: 2rem 0rem;
+    max-width: 78rem;
+  }
+
+  h1 {
+    width: 50rem;
+  }
+
+  .hero__img {
+    width: 70rem;
+    height: 34rem;
+  }
+
+  .content {
+    gap: 1rem 5rem;
+  }
+
+  .title {
+    width: 26.5rem;
+  }
+
+  .description {
+    width: 26.5rem;
+  }
+
+  .content__img {
+    width: 38.5rem;
+    height: 31.5rem;
+  }
+}
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  .nav__container {
+    padding: 1.3rem 3.2rem;
+  }
+
+  .nav__logo {
+    width: 10rem;
+  }
+
+  .btn__sign-in {
+    width: 10rem;
+    padding: 1.2rem 1.6rem;
+  }
+
+  .hero {
+    gap: 2.4rem;
+    padding: 11rem 3.2rem 0rem 3.2rem;
```

소소하게 아래처럼 축약해서 사용도 가능할거 같스빈다!

```suggestion
    padding: 11rem 3.2rem 0rem;
```

---

추가로 아래 두 가지가 어떻게 다른지도 한번 확인해보시면 좋을거 같아요!

```
1) padding: 11rem 3.2rem 0rem;
2) padding: 11rem 3.2rem 0;
```